gui/src/i18n/fr/translation.json

```diff
@@ -4,15 +4,15 @@
     "connection-lost": "Connexion avec le serveur perdue. Reconnexion..."
   },
   "tips": {
-    "find-tracker": "Impossible de différencier vos capteurs? Bougez-les pour qu'ils brillent à l'écran.",
-    "do-not-move-heels": "Assurez-vous de ne pas bouger vos pieds pendant la calibration!"
+    "find-tracker": "Impossible de différencier vos capteurs ? Secouez-en un pour qu'il soit mis en avant.",
+    "do-not-move-heels": "Assurez-vous de ne pas bouger vos pieds pendant la calibration !"
   },
   "body-part": {
     "NONE": "Non-assigné",
     "HEAD": "Tête",
     "NECK": "Cou",
     "RIGHT_SHOULDER": "Épaule droite",
-    "RIGHT_UPPER_ARM": "Bras supérieur droit",
+    "RIGHT_UPPER_ARM": "Bras droit",
```

Comment:
Why remove "supérieur?"
just "Bras" can be confusing and lead to think it's the whole arm...

Reply:
Because "bras supérieur" does not exist; "bras" is its name in french!

https://fr.wikipedia.org/wiki/Bras_(segment_de_membre)

Reply:
I wouldn't mind "Arrière-bras" though, if you'd prefer explicitness

Reply:
hmm idek what "Arrière-bras" means (except if it's the back of the arm?), and I doubt other people would.
I know "Bras" is supposed to mean that (in English as well I believe), but "Bras supérieur" isn't a word, so it doesn't have to be part of the dictionary, does it?

Reply:
solved on Discord, we'll use Bras and Avant-bras

Reply:
![image](https://user-images.githubusercontent.com/17962248/208203704-ef29d407-ad29-4b47-9247-a3af840db5fd.png)